src/core/qgsvectorlayerfeaturecounter.h

```diff
@@ -40,22 +42,44 @@ class CORE_EXPORT QgsVectorLayerFeatureCounter : public QgsTask
      */
     QgsVectorLayerFeatureCounter( QgsVectorLayer *layer, const QgsExpressionContext &context = QgsExpressionContext() );
 
+
+    /**
+     * Calculate the feature count and Ids per symbol
+     */
     bool run() override;
 
     /**
-     * Gets the count for each symbol. Only valid after the symbolsCounted()
+     * Returns the count for each symbol. Only valid after the symbolsCounted()
      * signal has been emitted.
      *
      * \note Not available in Python bindings.
      */
     QHash<QString, long> symbolFeatureCountMap() const SIP_SKIP;
 
     /**
-     * Gets the feature count for a particular \a legendKey.
+     * Returns the feature count for a particular \a legendKey.
      * If the key has not been found, -1 will be returned.
      */
     long featureCount( const QString &legendKey ) const;
 
+    /**
+     * Returns the QgsFeatureIds for each symbol. Only valid after the symbolsCounted()
+     * signal has been emitted.
```

Comment:
Maybe a `\see` linking the two methods would be good since they do (almost) the same?

Reply:
done